### PR TITLE
Fix block style paths for Windows

### PR DESCRIPTION
**Pull request by joemcgill**
This PR implements [the patch](https://core.trac.wordpress.org/attachment/ticket/59489/patch.diff) provided by [wildworks](https://profiles.wordpress.org/wildworks) on the original ticket.

Trac ticket: https://core.trac.wordpress.org/ticket/59489

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/blocks/index.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ function register_core_block_style_handles() {
 		$files = glob( wp_normalize_path( BLOCKS_PATH . '**/**.css' ) );
 		$files = array_map(
 			static function ( $file ) {
-				return str_replace( BLOCKS_PATH, '', $file );
+				return str_replace( wp_normalize_path( BLOCKS_PATH ), '', $file );
```
Comment (joemcgill) [inline src/wp-includes/blocks/index.php]: I think we can optimize this a bit by normalizing `BLOCKS_PATH` outside this `array_map` callback.

Comment (joemcgill) [reply src/wp-includes/blocks/index.php]: Addressed in [993934f](https://github.com/WordPress/wordpress-develop/pull/5409/commits/993934f8758655ad4482741d06a69fdb9869371d).

**Review by felixarntz — COMMENTED** [review]
@joemcgill The patch looks good to me.

Is it possible to add a unit test for this? I saw you mentioned that on the ticket, but looking at the code, I'm not sure this is testable given it uses a constant we can't override. I guess running the existing WordPress core unit tests on a Windows machine would be enough to uncover this bug?

**Comment by joemcgill** [comment]
Thanks @felixarntz. I do think that running the test suite on a windows OS would be the best way to uncover this type of issue. I've followed up with @desrosj and may open another ticket to address that issue. Otherwise, we'd need to refactor this function in a way that we could test it with mocked paths.

**Review by pbiron — COMMENTED** [review]
Good call on moving the normalization outside the loop!  And yes, it still works that way.

**Review by felixarntz — APPROVED** [review]
Given we cannot reasonably add unit tests for this (and running the existing tests on Windows would be sufficient to ensure this is fixed), I think this is good to commit.

**Comment by joemcgill** [comment]
Merged in https://core.trac.wordpress.org/changeset/56785

**Pull request closed without merging**